### PR TITLE
Changed IgnorePredicate to ignore all java. prefixed classes

### PR DESCRIPTION
**Pull request by cemnura**
Signed-off-by: Cem Nura <cem.nura@gmail.com>

### CODE REVIEW

**Review by phillip-kruger — APPROVED** [review]
LGTM.

**Review by gastaldi — APPROVED** [review]

**Comment by gsmet** [comment]
Any reason why you don't reuse the `ReflectiveHierarchyBuildItem#DefaultIgnorePredicate.INSTANCE`?

File: extensions/smallrye-graphql/deployment/src/main/java/io/quarkus/smallrye/graphql/deployment/SmallRyeGraphQLProcessor.java [hunk 1]
```diff
@@ -493,7 +493,7 @@ public void run() {
 
         @Override
         public boolean test(DotName t) {
-            return Classes.isPrimitive(t.toString()) || t.toString().startsWith("java.net.");
+            return Classes.isPrimitive(t.toString()) || t.toString().startsWith("java.");
```
Comment (gsmet) [inline extensions/smallrye-graphql/deployment/src/main/java/io/quarkus/smallrye/graphql/deployment/SmallRyeGraphQLProcessor.java:496]: And maybe we should handle the primitives in `DefaultIgnorePredicate` directly.

/cc @geoand

**Review by gsmet — CHANGES_REQUESTED** [review]
Added some suggestions, let's sort them out before merging.

File: extensions/smallrye-graphql/deployment/src/main/java/io/quarkus/smallrye/graphql/deployment/SmallRyeGraphQLProcessor.java (hunk 1 above)
Comment (geoand) [reply extensions/smallrye-graphql/deployment/src/main/java/io/quarkus/smallrye/graphql/deployment/SmallRyeGraphQLProcessor.java:496]: Makes sense

Comment (gastaldi) [reply extensions/smallrye-graphql/deployment/src/main/java/io/quarkus/smallrye/graphql/deployment/SmallRyeGraphQLProcessor.java:496]: Ah yes, good catch. That's a good idea

Comment (gastaldi) [reply extensions/smallrye-graphql/deployment/src/main/java/io/quarkus/smallrye/graphql/deployment/SmallRyeGraphQLProcessor.java:496]: I'll provide a PR

**Comment by gastaldi** [comment]
There you go: https://github.com/quarkusio/quarkus/pull/10019

File: extensions/smallrye-graphql/deployment/src/main/java/io/quarkus/smallrye/graphql/deployment/SmallRyeGraphQLProcessor.java (hunk 1 above)
Comment (cemnura) [reply extensions/smallrye-graphql/deployment/src/main/java/io/quarkus/smallrye/graphql/deployment/SmallRyeGraphQLProcessor.java:496]: Sounds like general solution.

**Comment by cemnura** [comment]
> Any reason why you don't reuse the `ReflectiveHierarchyBuildItem#DefaultIgnorePredicate.INSTANCE`?

No reason specifically. I thought @phillip-kruger may explain?

**Comment by gsmet** [comment]
Superseded by https://github.com/quarkusio/quarkus/pull/10019

**Pull request closed without merging**